src/west/commands/project.py

```diff
@@ -18,69 +20,213 @@
 from west import log
 from west import util
 from west.commands import WestCommand, CommandError
-from west.manifest import Manifest, MalformedManifest, MANIFEST_PROJECT_INDEX
+from west.manifest import Manifest, MalformedManifest, MalformedConfig, \
+    MANIFEST_PROJECT_INDEX
 from west.manifest import MANIFEST_REV_BRANCH as MANIFEST_REV
 from west.manifest import QUAL_MANIFEST_REV_BRANCH as QUAL_MANIFEST_REV
 from urllib.parse import urlparse
 import posixpath
 
 
-class PostInit(WestCommand):
+class Init(WestCommand):
```

Comment:
I wonder if this really belongs here. Maybe we should have a `commands/core.py` but let's leave this  for after

Reply:
IMO it's okay here. This is the file where we keep the multi-repo commands.